.github/workflows/nf-test-gpu.yml

```
@@ -96,7 +96,7 @@ jobs:
           tags: "gpu"
 
   confirm-pass:
```

I think this should be 
```suggestion
  confirm-pass-gpu:
```
to avoid false-positives (and update the branch protection rules)